src/main/java/net/datafaker/Unique.java

```diff
@@ -0,0 +1,72 @@
+package net.datafaker;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Unique extends AbstractProvider {
```

Comment:
Would it be possible to add a bit of documentation to the class and maybe public methods? It's a little light on documentation as it stands, which makes it a bit harder to use for some people

Reply:
I added some brief documentation to the class itself and the method. Let me know if anything can be phrased differently or if more explanation is necessary. I'll also work on getting some actual documentation added into the markdown docs.